services/mailer.js

```diff
@@ -13,6 +16,20 @@ if (smtpRequiredProps.some(prop => !process.env[prop])) {
   console.error(`${smtpRequiredProps.join(', ')} should be defined in the environment if you would like to send password reset emails from Talk`);
 }
 
+// load all the templates as strings
+const templateStrings = {};
+fs.readdir(path.join(__dirname, 'email'), (err, files) => {
+  if (err) {
+    throw err;
+  }
+
+  files.forEach(file => {
+    fs.readFile(path.join(__dirname, 'email', file), 'utf8', (err, data) => {
+      templateStrings[file] = _.template(data);
+    });
+  });
+});
+
```

Comment:
Should have provided live-reloading for development, and production loads them all in once. Might have also been better to abstract the access to the object to a function factory to do it for you:

```js
const template = {};
if (process.env.NODE_ENV === 'production') {

  // Returns the data loaded at application start.
  template.data = fs.readdirSync(path.join(__dirname, 'email')).reduce((data, filename) => {

    // Split the filename.
    let splitFileName = filename.split('.');

    // Remove the 'ejs' extension.
    splitFileName.splice(splitFileName.length - 1, 1);

    const [name, format] = splitFileName;

    if (!(name in data)) {
      data[name] = {};
    }

    data[name][format] = _.template(fs.readFileSync(filename, 'utf8'));
  }, {});

  // Renders a template based on the request name.
  template.render = (name, format = 'txt', context) => template.data[name][format](context);
} else {
  template.render = (name, format = 'txt', context) => new Promise((resolve, reject) => {
    fs.readFile(path.join(__dirname, 'email', [name, format, 'ejs'].join('.')), (err, file) => {
      if (err) {
        return reject(err);
      }

      return resolve(_.template(file)(context));
    });
  });
}
```

Which would make it consistent with the rest of the development behaviour around templates.